lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
```

Comment:
yeah i think you should just go ahead with removing it from the main GLI code. 👍